docs/en/ingest-management/elastic-agent/running-on-kubernetes-gke-standard-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,170 @@
+  value: "fleet-server-policy" <1>
+- name: KIBANA_HOST
+  value: ""
+- name: KIBANA_FLEET_USERNAME
+  value: ""
+- name: KIBANA_FLEET_PASSWORD
+  value: ""
+------------------------------------------------
+<1> To learn how to create a policy, refer <<create-a-policy-no-ui>>.
+
+
+// Begin collapsed section
+[%collapsible]
+.Configuration details
+====
+****
+
+[cols="2*<a"]
+|===
+| Settings | Description
+
+include::configuration/env/shared-env.asciidoc[tag=fleet-url]
+
+include::configuration/env/shared-env.asciidoc[tag=fleet-enrollment-token]
+
+include::configuration/env/shared-env.asciidoc[tag=fleet-server-policy-id]
+
+include::configuration/env/shared-env.asciidoc[tag=kibana-host]
+
+include::configuration/env/shared-env.asciidoc[tag=kibana-fleet-username]
+
+include::configuration/env/shared-env.asciidoc[tag=kibana-fleet-password]
+|===
+
+Refer to <<agent-environment-variables>> for all available options.
+
+****
+====
+
+
+[discrete]
+== Deploy
+
+If planning to deploy `state_*` datasets of Kubernetes package,
+https://github.com/kubernetes/kube-state-metrics#usage[kube-state-metrics] needs to be already deployed
+in the cluster. If `kube-state-metrics` is not already running, deploy it now (see the
+https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment[Kubernetes
+deployment] docs).
+
+To deploy {agent} on Kubernetes, run:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+kubectl create -f elastic-agent-managed-kubernetes.yaml
+------------------------------------------------
+
+To check the status, run:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+$ kubectl get pod -n kube-system -l app=elastic-agent
+
+NAME                  READY   STATUS    RESTARTS   AGE
+elastic-agent-hrjbg   1/1     Running   0          12m
+elastic-agent-olpsd   1/1     Running   0          12m
+------------------------------------------------
+
+NOTE: You might need to adjust https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[resource limits] of the elastic-agent container
+in the `elastic-agent-managed-kubernetes.yaml` manifest. Container resource usage depends on the number of data streams
+and the environment size.
+
+{agent}s should be enrolled to {fleet} and users should be able to deploy the Kubernetes package accordingly.
+This can be confirmed in {kib} under {fleet}  / Agents section.
+
+
+[discrete]
+== Deploying {agent} to collect cluster-level metrics in large clusters
+
+The size and the number of nodes in a Kubernetes cluster can be fairly large at times,
+and in such cases the Pod that will be collecting cluster level metrics might face performance
+issues due to resources limitations. In this case users might consider to avoid using the
+leader election strategy and instead run a dedicated, standalone {agent} instance using
+a Deployment in addition to the DaemonSet.
+
+[discrete]
+== Not available data
+
```

Comment:
Can we add here or to another section details for the orchestratot.cloud.name with add_field ?
Just an example oof the processor with one liner to explain what problem it solves

Also for autopilot: GKE autopilot installations do now allow agent installation. At the moment several changes are needed to the provided manifest in order to be installed. Do we plan to have another version of such manifest online?

Reply:
This PR is only for GKE standard. Autopilot will be covered separately once we have all the details figured out regarding the requirements etc.